03.1/loaders/controller-logger/index.ts

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs';
+import path from 'path';
+
+import { Application, Request } from 'express';
+import morgan from 'morgan';
+
+// create a write stream (in append mode)
+const errorLogStream = fs.createWriteStream(path.join(process.cwd(), 'controller-logs.log'), {
+  flags: 'a',
+});
+
+morgan.token('args', (req: Request): string => {
+  if (req.body && Object.keys(req.body).length) {
+    return JSON.stringify(req.body);
```

Comment:
Lets avoid using password in logs (better to use '***')

Reply:
thanks, updated.